nodejs/week1/MealWebsite/src/backend/routes/meals.js

```diff
@@ -0,0 +1,19 @@
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+let mealReviews = []; // empty array to push reviews
+
+meals.forEach((meal) => {
+  reviews.forEach((review) => {
+    if (meal.id == review.mealId) {
+      mealReviews.push(review);
+    }
+    if (mealReviews.length == 0) {
+      // if there are no reviews for a meal
+      meal["reviews"] = ["NO reviews"];
+    } else {
+      meal["reviews"] = mealReviews;
+    }
+  });
+});
```

Comment:
You have a bug at `/meals` route. ❕ 
Check which meals show which reviews. 

Try to avoid `.forEach`. You can do a `for` loop over `meals`, but avoid nesting `for` loops if you can avoid it (and you can!). 
You can instead use `mealReviews.filter(...)` to filter out the reviews which have `mealId` equal to `meal[i].id` (where `i` is the index of the current meal). 

Also, when there are no reviews, just return an empty array.